refinery/core/tests.py

```diff
@@ -1593,9 +1589,23 @@ def setUp(self):
 
         self.factory = APIRequestFactory()
         self.client = APIClient()
-        self.view = DataSetsViewSet.as_view()
+        self.url_root = '/api/v2/{}'.format(kwargs.get("api_base_name"))
+        self.view = kwargs.get("view")
 
-        self.url_root = '/api/v2/data_sets/'
+        self.client.login(username=self.username, password=self.password)
+
+
+class DataSetApiV2Tests(APIV2TestCase):
+    def create_rand_str(self, count):
+        return ''.join(
+            random.choice(string.ascii_lowercase) for _ in xrange(count)
+        )
```

Comment:
Probably better not to introduce non-deterministic behavior?

Reply:
I think this just moved up/down in the diff. I didn't introduce it. I'll look into it though now that I'm around it.